ngx-tools/src/retry-with-backoff/retry-with-backoff.spec.ts

```diff
@@ -14,6 +14,7 @@ describe(`retryWithBackoff`, function() {
     const error = new Error('bar');
     const seenValues: {[idx: number]: number} = {};
 
+    // tslint:disable-next-line deprecation
```

Comment:
We're actually just getting around a TSLint bug. The deprecation rule is triggered even though `of` itself is not deprecated; only certain overloads are. We did this a bit here also https://github.com/GetTerminus/terminus-ui/pull/1488

There is an open PR to fix this which would allow us to remove all of these disables. But since TSLint itself is deprecated I'm not holding my breath on it getting fixed.